modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/outgoing/OutgoingSnapshot.java

```diff
@@ -71,18 +76,23 @@ public class OutgoingSnapshot {
 
     private final ReusableLockLockup mvOperationsLockup = new ReusableLockLockup(mvOperationsLock);
 
+    /**
+     * Snapshot metadata taken on snapshot scope freezing.
+     */
+    private SnapshotMeta meta;
```

Comment:
I suggest renaming it.

Reply:
How? `snapshotMeta`?

Reply:
To make it clear that this field will be set on freeze.

Reply:
Maybe **metaAfterFreezee** ?

Reply:
Renamed it to `frozenMeta`